modules/libcom/test/Makefile

```diff
@@ -339,6 +339,20 @@ TESTS += nonEpicsThreadPriorityTest
 endif
 endif
 
+# HK: improve detection of valid command line libraries
+ifeq ($(COMMANDLINE_LIBRARY),READLINE)
+TESTPROD_HOST += readlineTest
+readlineTest_SRCS += readlineTest.c
+TESTS += readlineTest
+TESTFILES += ../multiline-input.txt ../multiline-expect.txt
+endif
```

Comment:
Core Group: Instead of duplicating these `ifeq()` blocks, please use this Makefile conditional which uses a list of acceptable names:

```Makefile
ifneq ($(filter READLINE EPICS,$(COMMANDLINE_LIBRARY)),)
TESTPROD_HOST += readlineTest
...
endif
```

Some older linux OSs needed additional choices for `COMMANDLINE_LIBRARY` to adjust the OS libraries and compiler flags, using the names `READLINE_CURSES` and `READLINE_NCURSES`. Our Cygwin support also used yet another name, but we don't support Cygwin any more. However our more recent automatic readline discovery process doesn't support these so it's likely they aren't required any more, and I wouldn't bother adding them here (any user who manually configures those settings just doesn't run this self-test).